### PR TITLE
Fix namespace, add strict_types

### PR DESCRIPTION
**Pull request by alexkart**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  |

### CODE REVIEW

File: src/WinCache.php [hunk 1]
```diff
@@ -75,9 +75,9 @@ public function has($key): bool
     /**
      * @noinspection PhpDocMissingThrowsInspection DateTime won't throw exception because constant string is passed as time
      *
-     * Normalizes cache TTL handling `null` value and {@see DateInterval} objects.
-     * @param int|DateInterval|null $ttl raw TTL.
-     * @return int|null TTL value as UNIX timestamp or null meaning infinity
+     * Normalizes cache TTL handling `null` value, strings and {@see DateInterval} objects.
+     * @param int|string|DateInterval|null $ttl raw TTL.
```
Comment (samdark) [inline src/WinCache.php]: Why do we want strings there?

Comment (alexkart) [reply src/WinCache.php]: Although PSR interface has doc blocks `@param null|int|\DateInterval $ttl` for the methods, but it cannot be type hinted. So, a string can be also passed.
Alternatively, we can throw `InvalidArgumentException` if a string is passed.

**Comment by alexkart** [comment]
@samdark could you please cancel this hung build https://ci.appveyor.com/project/samdark/cache-wincache/builds/26693625 I've fixed it.

**Comment by samdark** [comment]
Done.

File: src/WinCache.php (hunk 1 above)
Comment (samdark) [reply src/WinCache.php]: Casting should be fine.

**Pull request merged**